azurerm/resource_arm_automation_account.go

```diff
@@ -59,6 +61,17 @@ func resourceArmAutomationAccount() *schema.Resource {
 				},
 			},
 
+			"sku_name": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
For new properties we shouldn't be ignoring case without good reason such as a broken API, can we remove this?
```suggestion
```

Reply:
Sure...